stackstate_checks_base/tests/test_agent_check_v2.py

```diff
@@ -0,0 +1,1610 @@
+# -*- coding: utf-8 -*-
+
+# (C) StackState, Inc. 2022
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import copy
+import json
+import mock
+import pytest
+import sys
+
+from six import PY3, text_type
+from schematics import Model
+from schematics.exceptions import ValidationError, ConversionError, DataError
+from schematics.types import URLType, ListType, StringType, IntType, BooleanType, ModelType
+
+from stackstate_checks.checks import AgentCheckV2, TransactionalAgentCheck, StatefulAgentCheck, StackPackInstance, \
+    TopologyInstance, AgentIntegrationInstance, HealthStream, HealthStreamUrn, Health, CheckResponse
+from stackstate_checks.base.stubs import datadog_agent
+from stackstate_checks.base.stubs.topology import component
+from stackstate_checks.base.utils.state_api import generate_state_key
+
+
+TEST_INSTANCE = {
+    "url": "https://example.org/api"
+}
+
+
+def test_instance():
+    """
+    Simply assert the class can be instantiated
+    """
+    AgentCheckV2()
```

Comment:
Why is this here? Its not assigned to variable.

Reply:
No idea, was inherited from `test_agent_check` so I'll remove it 😄